src/sentry/conf/types/consumer_definition.py

```diff
@@ -5,11 +5,17 @@
 
 import click
 
+from sentry.conf.types.topic_definition import Topic
+
 
 class ConsumerDefinition(TypedDict, total=False):
     # Which logical topic from settings to use.
-    topic: Required[str | Callable[[], str]]
-    default_topic: str
+    topic: Topic
+
+    # Override topic. To be deprecated
+    topic_override: Required[str | Callable[[], str]]
+
+    validate_schema: str
```

Comment:
Is this the name of the schema in the schema registry ?

Reply:
this is a typo - it's supposed to be a bool